src/components/common/MaintenanceBar.vue

```diff
@@ -49,6 +75,18 @@ export default {
   color: var(--bright);
 }
 
+.maintenance-bar.ok {
+  background-color: #61b360;
```

Comment:
please use colors from variables that already exist or add the colors to the variables. This way we can have an overview about used colors.